integrations/flow-judge.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: integration
+name: Flow Judge
+description: Evaluate Haystack pipelines using Flow Judge
+authors:
+    - name: Flow AI
+      socials:
+        github: flowaicom
+        twitter: flowaicom
+        linkedin: https://www.linkedin.com/company/flowaicom/ 
+pypi: https://pypi.org/project/flow-judge/
+repo: https://github.com/flowaicom/flow-judge
+type: Evaluation Framework
+report_issue: https://github.com/flowaicom/flow-judge/issues
+logo: /logos/flow-ai.png
+version: Haystack 2.0
+toc: true
+---
+### **Table of Contents**
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+- [License](#license)
+
+## Overview
+This integration allows you to evaluate Haystack pipelines using Flow Judge.
+
+Flow Judge is an open-source, lightweight (3.8B) language model optimized for LLM system evaluations. Crafted for accuracy, speed, and customization.
+
+Read the technical report [here](https://www.flow-ai.com/blog/flow-judge).
+
+## Installation
+
+For running Flow Judge with vLLM engine:
+```bash
+pip install flow-judge[vllm]
+pip install 'flash_attn>=2.6.3' --no-build-isolation
+```
+For running Flow Judge with transformers:
+```bash
+pip install flow-judge[hf]
+```
+If flash attention:
+```bash
+pip install 'flash_attn>=2.6.3' --no-build-isolation
+```
+For running Flow Judge with Llamafile on macOS:
+```bash
+pip install flow-judge[llamafile]
+pip install 'flash_attn>=2.6.3' --no-build-isolation
+```
+
```

Comment:
It would be great to share a documentation page here. Something around this:
```suggestion
To learn more about the installation, visit the [Flow Judge Installation](url) page.
```